core/src/main/java/com/hcl/ecomm/core/models/RecaptchaSiteKey.java

```diff
@@ -0,0 +1,34 @@
+package com.hcl.ecomm.core.models;
+
+import com.hcl.ecomm.core.services.CustomerService;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.models.annotations.Model;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+
+@Model(adaptables = Resource.class)
+public class RecaptchaSiteKey {
```

Comment:
@nitheshkumar-hcl  rename this and append Model as this sling model class.

Reply:
Committed the changes as per the review comments